### PR TITLE
[Bug 1061546] Suggestions 

### PR DESCRIPTION
**Pull request by Rik**

### CODE REVIEW

File: app/js/question_controller.js [hunk 1]
```diff
@@ -225,10 +226,55 @@
...
+    panels.forEach(function(panel) {
+      if (panel == elt) {
+        panel.classList.remove('hide');
+      } else {
+        panel.classList.add('hide');
+      }
+    });
+  }
+
+  function give_suggetions(evt) {
```
Comment (mythmon) [inline app/js/question_controller.js]: This function name is misspelled. It should be `give_suggestions`.

Comment (Rik) [inline app/js/question_controller.js]: Thanks, fixed.

File: app/js/sumo_db.js [hunk 2]
```diff
@@ -165,7 +172,7 @@
 
     get_active_questions: function(username) {
       var endpoint = API_V2_BASE + 'question/';
-      endpoint += '?involved=' + username
+      endpoint += '?involved=' + username;
```
Comment (rehandalal) [inline app/js/sumo_db.js]: oops! my bad.

**Comment by rehandalal** [comment]
lgtm. i think there's probably a better way to handle the race condition stuff but that will likely require some changes on the server side. for now this is good!

**Pull request closed without merging**